src/directLine.ts

```diff
@@ -9,6 +9,7 @@ import { IScheduler } from 'rxjs/Scheduler';
 import { Subscriber } from 'rxjs/Subscriber';
 import { Subscription } from 'rxjs/Subscription';
 import { async as AsyncScheduler } from 'rxjs/scheduler/async';
+import { decode } from 'jsonwebtoken';
```

Comment:
Instead of using [`jsonwebtoken` package](https://npmjs.com/package/jsonwebtoken), please use [`jwt-decode` package](https://www.npmjs.com/package/jwt-decode) instead
- `jsonwebtoken` package only works under Node.js but not browser, because it depends on Node.js `crypto` library
- Webpack 5 will no longer polyfill `crypto` library in browser (`node-libs-browser` will no longer be included)
- Since we are using `jsonwebtoken` to parse and not verify the token, it's okay to replace it with `jwt-decode`